bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/MichellePrinter.java

```diff
@@ -112,12 +112,15 @@ private void writeImagingOutput(final ImagingResult result, final ItcImagingResu
         } else {
             _println("Polarimetry mode enabled.\n");
             final String result2 = CalculatablePrinter.getTextResult(result.is2nCalc(), result.observation());
-            final String delims = "[ ]+";
+            final String delims = "[ ,]+";
             final String[] tokens = result2.split(delims);
             for (int i = 0; i < tokens.length; i++) {
                 if (tokens[i].contains("Derived")) {
                     tokens[i + 5] = String.format("%.2f", new Double(tokens[i + 5]) * 4);
-                    tokens[i + 9] = String.format("%.2f", new Double(tokens[i + 9]) * 4);
+                    if (tokens[i + 6].contains("each"))
+                        tokens[i + 12] = String.format("%.2f", new Double(tokens[i + 12]) * 4);
+                    else
+                        tokens[i + 8] = String.format("%.2f", new Double(tokens[i + 8]) * 4);
```

Comment:
Wow is there any documentation about what we expect to see at these offsets? This kind of code is really fragile.